deps/Boost/Boost.cmake

```diff
@@ -147,8 +147,8 @@ if (NOT IS_CROSS_COMPILE OR NOT APPLE OR BUILD_SHARED_LIBS)
     message(STATUS "Standard boost build with install command '${_install_cmd}'")
 ExternalProject_Add(
     dep_Boost
-   URL "https://boostorg.jfrog.io/artifactory/main/release/1.75.0/source/boost_1_75_0.tar.gz"
-    URL_HASH SHA256=aeb26f80e80945e82ee93e5939baebdca47b9dee80a07d3144be1e1a6a66dd6a
+   URL "https://boostorg.jfrog.io/artifactory/main/release/1.78.0/source/boost_1_78_0.tar.gz"
```

Comment:
Hehe, I had trouble cross-compiling OrcaSlicer for Mac, which was required for CICD since there is no M1 runner. That's why I kept the Boost library at 1.75

But I think we can upgrade it only for Windows first.
Can you make the change for Windows only?

Reply:
Sure thing!